wordpress-mobile/gutenberg-mobile
# test: Add jest-watch-typeahead

Simplify scoping test runs in watch mode. This addition mirrors an
already completed addition in Gutenberg proper. Relates to https://github.com/WordPress/gutenberg/pull/51869.

To test: 

1. Enable Jest watch mode: `npm run test -- --watch`
2. Press <kbd>p</kbd> to enable test file name filtering. 
3. Type to begin filtering, select a test file. 
4. Verify only the selected file runs. 
5. Press <kbd>t</kbd> to enable test name filtering. 
6. Type to begin filtering, select a test. 
7. Verify only the selected test runs. 

PR submission checklist:
